File: src/main/java/org/pixeltime/enchantmentsenhance/interfaces/PlaySound.java
package org.pixeltime.enchantmentsenhance.interfaces;

import org.bukkit.entity.Player;

public interface PlaySound {
    /**
     * Plays a sound to a player.
     *
     * @param p
     * @param type
     */
     void playSound(Player p, String type);
}

File: modules/EnchantmentsEnhance-Plugin/src/main/kotlin/org/pixeltime/enchantmentsenhance/interfaces/PlaySound.kt
package org.pixeltime.enchantmentsenhance.interfaces

import org.bukkit.entity.Player

interface PlaySound {
    /**
     * Plays a sound to a player.
     *
     * @param p
     * @param type
     */
    fun playSound(p: Player, type: String)
}
